test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -3354,9 +3390,17 @@ class PBSObject(object):
     :type defaults: Dictionary
     """
 
+    logger = logging.getLogger(__name__)
     utils = BatchUtils()
     platform = sys.platform
 
+    ptl_conf = {
+        'mode': PTL_API,
+        'max_attempts': 60,
+        'attempt_interval': 0.5,
```

Comment:
Should these values be set based on object.options['PTL_MAX_ATTEMPTS'] and object.options['PTL_ATTEMPT_INTERVAL'] as opposed to hardcoding them? I noticed that the same values are defined in two places.

Reply:
These are default values if a user or any PTL command doesn't call `PtlConfig()` and options dict is only accessible in PtlConfig class not in PBSObject or its child

Reply:
@mike0042 Sorry for pushing but this PR is little urgent for other dependent work so if you don't have any further comment then can you please do needfull...